pages/[state]/[district]/medicine.js

```diff
@@ -16,29 +19,64 @@ export default function Medicine({ state, district, medicineByDistrict }) {
         list={[
           { href: `/${state}`, name: humanize(state) },
           { href: `/${state}/${district}`, name: humanize(district) },
-          { href: null, name: 'Medicines' },
+          { href: null, name: "Medicines" },
         ]}
       />
-      <h1 className='mt-4 font-black text-6xl text-gray-900 md:text-left text-center'>
+      <h1 className="mt-4 font-black text-6xl text-gray-900 md:text-left text-center">
         {humanize(district)}
       </h1>
-      <div className='space-y-4 mt-4'>
+      <div className="space-y-4 mt-4 mb-4">
         {medicineByDistrict.map((p) => {
           return (
-            <div
-              key={p.id}
-              className='bg-white p-4 flex shadow rounded-lg justify-between'
-            >
-              <div>
-                <div className='font-bold'>{p.name}</div>
-                <div className='font-bold'>{p.email}</div>
-                <div>{p.description}</div>
-                <div>{p.address}</div>
-                <div>{parseDateString(p.createdTime)}</div>
+            <div className="bg-white rounded-lg shadow">
+              <div key={p.id} className="p-4 flex justify-between">
+                <div>
+                  <div className="font-bold text-2xl">
+                    {p.name}
+                    <div className="text-sm text-gray-700 font-semibold">
+                      <span className="mr-2">{p.distributorName}</span>|
+                      <span className="ml-2">{p.city}</span>
+                    </div>
+                  </div>
+                  <div className="w-11/12 max-w-3xl mt-2">
+                    <div className="text-sm">{p.address}</div>
+                  </div>
+                </div>
+                <div className="flex space-x-2 items-start">
+                  {p.phone1 && (
+                    <a
+                      className="font-mono text-gray-800 hover:text-gray-900 text-sm font-bold"
+                      href={`tel:${p.phone1}`}
+                    >
+                      <FontAwesomeIcon
+                        title={p.phone1}
+                        className="text-xl w-6"
+                        icon={faPhone}
```

Comment:
<img width="126" alt="Screenshot 2021-04-22 at 8 02 52 PM" src="https://user-images.githubusercontent.com/61158210/115732579-ba921800-a3a5-11eb-9651-610ed2038b89.png">

Instead of `faPhone` can you use `faPhoneAlt`

Also it goes along with the Helpline logo

<img width="105" alt="Screenshot 2021-04-22 at 8 04 46 PM" src="https://user-images.githubusercontent.com/61158210/115732866-ff1db380-a3a5-11eb-95ab-4d0ad2473896.png">


I think the "Mirrored Icon" is more descriptive.

Reply:
yep, already done pushing in some time.

Reply:
the icons seem to be behaving strangely sometimes

Reply:
yep i think font-awesome is the one causing this [Bloated CSS problem](https://life.coronasafe.network/_next/static/css/491fd2130f24e2c92500.css)

Reply:
@Deep-Codes take a look, pushed the fixes.
![image](https://user-images.githubusercontent.com/50803990/115740102-1fe90780-a3ac-11eb-8bb8-33f60b915c2e.png)